### PR TITLE
[MOC-198] find and replace

### PR DESCRIPTION
**Pull request by fitzk**
Related PR: https://github.com/Mocksi/nest/pull/22

Screenshot 2024-08-22 at 12 17 42 PM

## Summary by CodeRabbit

- **New Features**
	- Enhanced messaging structure in the Chrome extension with added context in the responses.
	- Introduced dynamic text replacement functionality within the Iframe component, triggered by specific messages.
- **Improvements**
	- Improved modularity and maintainability of the code for better handling of text replacements.
- **Configuration**
	- Established a robust Vite configuration for developing and building the Chrome extension, integrating necessary plugins and settings for both development and production environments.

### CODE REVIEW

File: apps/mocksi-lite-next/vite.config.ts.timestamp-1724351457304-9b25862a7b76b.mjs
Comment (jonathankap) [inline apps/mocksi-lite-next/vite.config.ts.timestamp-1724351457304-9b25862a7b76b.mjs:1]: I think this is an internal file that shouldn't be checked in.

File: apps/mocksi-lite-next/vite.config.ts.timestamp-1724351552357-a114ea3b305ca.mjs
Comment (jonathankap) [inline apps/mocksi-lite-next/vite.config.ts.timestamp-1724351552357-a114ea3b305ca.mjs:1]: Also remove from commit

**Review by jonathankap — CHANGES_REQUESTED** [review]
Please remove the two extra files

File: apps/mocksi-lite-next/vite.config.ts.timestamp-1724351552357-a114ea3b305ca.mjs
Comment (fitzk) [reply apps/mocksi-lite-next/vite.config.ts.timestamp-1724351552357-a114ea3b305ca.mjs:1]: yes for sure, just pushed up another PR that adds them to .gitinore too

**Review by jonathankap — APPROVED** [review]
Looks good

**Pull request merged**